$MirOS$

	Since fixing up -no-undefined in libtool, this fails.
	No idea why but, since this is bootstrap-only tool,
	not bothering looking for it either.

--- libssp/Makefile.am.orig	Thu Aug 21 11:58:17 2008
+++ libssp/Makefile.am	Sun Dec  1 21:08:50 2013
@@ -35,7 +35,7 @@ libssp_la_SOURCES = \
 libssp_la_LIBADD = 
 libssp_la_DEPENDENCIES = $(version_dep) $(libssp_la_LIBADD)
 libssp_la_LDFLAGS = -version-info `grep -v '^\#' $(srcdir)/libtool-version` \
-		    $(version_arg) -no-undefined
+		    $(version_arg)
 
 libssp_nonshared_la_SOURCES = \
 	ssp-local.c
